tests/test_table.py

```diff
@@ -1426,8 +1426,7 @@ def test_type():
 def test_update(table, overwrite, others):
     df = table.get()
     table.update(others, overwrite=overwrite)
-    if isinstance(others, audformat.Table):
-        others = [others]
+    others = audeer.to_list(others)
```

Comment:
What does this change do?

Reply:
Just a simplification to ensure `others` is a list or what do you mean?

Reply:
I mean the removal of `if isinstance(others, audformat.Table):`

Reply:
Ah, sorry, no I get it.